config/settings.yml

```diff
@@ -0,0 +1,12 @@
+dor_services:
+  url: 'https://fakeuser:fakepass@fakeserver.example.edu/dor'
+workflow:
+  url: 'https://workflows.example.org'
+  # logfile: 'log/wfs/workflow_service.log'
+  # shift_age: 'weekly'
+  timeout: 60
+transfer_object:
+  dor_host: 'userid@dor-host'
+  input_dir: '/dor/export/'
```

Comment:
i believe these were intentionally renamed (to `from_host` and `from_dir`) by @ndushay in some of the work that this PR got rebuilt on top of (compared to the version of the code the work in this PR was originally built on, which used `dor_host` and `input_dir`, IIRC).

Reply:
Not really attached to `input_dir`, but `dor_host` seems a lot more descriptive than `from_host`.  My main concern is just that the configs match the code, which they do here.  If you want different keys, just tell me what to use.

Reply:
yeah - the configs changed between the orig PR and this re-do.  I will add a commit to fix this.  "dor_host" is also misleading because the files are created by common_accessioning robot, on a lyberservices box.  Yay us.